### ISSUE TITLE
Interpolation broken (again?)

### ISSUE THREAD

**Issue by pixelass** [sasstools/sass-lint#819]
String interpolations cause sass-lint to fail. Due to this issue I have to disable sass-lint entirely.

* "sass-lint": "1.8.2"

* default config as in example

```scss
.#{$prefix}-0 {
    display: none;
}
```

* The selector should be resolved

* Instead I get an error

```shell
57:1  error  Please check validity of the block starting from line #57  Fatal
```

I used CLI

**Comment by bgriffith** [comment]
Hey @pixelass - thanks for the report! We've already noted this and I've posted a PR (https://github.com/tonyganch/gonzales-pe/pull/208) so hopefully this will be fixed in the next few days.

**Comment by Arial7** [comment]
This got Fixed in Gonzales-Pe 3.4.4.

**Comment by DanPurdy** [comment]
@Arial7 We've already pushed gonzales-pe 3.4.4 into our develop branch with #821 It will be available with our next release.

**Comment by Arial7** [comment]
Yes, I know, I just wanted you to close the Issue 😉

**Comment by DanPurdy** [comment]
ah, 😄

**Comment by pixelass** [comment]
Sorry, my bad.. should have tagged it as duplicate.. 

Thx.
